### PR TITLE
Dac adc update

### PR DESCRIPTION
**Pull request by ckometter**
-Stop ramp does not produce an infinite loop.
-Disseminated buffer ramp compatible with new firmware. https://github.com/afylab/Electronics/pull/4

### CODE REVIEW

File: dac_adc.py [hunk 1]
```diff
@@ -277,10 +284,12 @@ def buffer_ramp(self,c,dacPorts,adcPorts,ivoltages,fvoltages,steps,delay,nReadin
         voltages = []
         channels = []
         data = ''
+
+        dev.ramping(True)
```
Comment (ckometter) [inline dac_adc.py]: I did not place a yield because It's a return and not a returnValue, but I'm not sure

**Comment by mserlin** [comment]
It's on my todo list to check the functionality. Carlos and I have been communicating, just haven't had a chance to do it yet

**Comment by mserlin** [comment]
Everything works (with minor exception noted below) as far as I can tell with the latest version I uploaded. Testing / confirmation from someone else would be nice. 

There is the very minor issue that, when you use the stop_ramp function, one voltage data point is not read properly. This is because the arduino sends the voltage data via serial to the computer during the conversion of the next voltage. This means that, if you interrupt the ramp, that last data point doesn't get read. There's probably a way to fix this that I couldn't think of, but it probably doesn't really matter...

**Comment by mserlin** [comment]
Weird, I don't know how that happened. Anyways, I think I just changed 2 things. 

(1) I changed the time.sleep(1) command for one that works asynchronously in the stop_ramp function. 
(2) Made it so that, any values that were not read during a buffer ramp are filled with 0s.

**Pull request merged**